src/main/java/org/sagebionetworks/web/client/widget/entity/controller/ProvenanceEditorWidget.java

```diff
@@ -80,41 +80,43 @@ public void onSuccess(Activity result) {
 				view.setName(result.getName());
 				view.setDescription(result.getDescription());
 				Set<Used> allProvenance = result.getUsed();
-				List<ProvenanceEntry> usedEntries = new LinkedList<ProvenanceEntry>();
-				List<ProvenanceEntry> executedEntries = new LinkedList<ProvenanceEntry>();
-				for (Used provEntry: allProvenance) {    
-					ProvenanceEntry toAdd;
-					if (provEntry instanceof UsedEntity) {
-						Reference ref = ((UsedEntity)provEntry).getReference();
-						toAdd = ginInjector.getEntityRefEntry();
-						String entityId = ref.getTargetId();
-						Long version = ref.getTargetVersionNumber();
-						String versionString = null;
-						if (version != null) {
-							versionString = version.toString();
+				if (allProvenance != null) {
```

Comment:
Please add a test to ProvenanceEditorWidgetTest that returns null for the returned Activity getUsed()